applications/osb-portal/src/components/workspace/drawer/WorkspaceDrawer.tsx

```diff
@@ -89,9 +89,10 @@ interface WorkspaceDrawerProps {
   workspace: Workspace;
   user: UserInfo;
   refreshWorkspace: () => any;
+  cloneWorkspace?: (wsId: number) => any,
 }
 
-export const WorkspaceDrawer: React.FunctionComponent<WorkspaceDrawerProps> = ({ user, children, workspace }) => {
+export const WorkspaceDrawer: React.FunctionComponent<WorkspaceDrawerProps> = ({ user, children, workspace, cloneWorkspace }) => {
```

Comment:
See comment below: it should not be needed to pass the cloneWorskpace action explicitly here